### DISCUSSION TITLE
thanhle.blog/en/blog/bay-gio-hoc-blockchain-lieu-co-muon

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#40] [Ideas]
# thanhle.blog/en/blog/bay-gio-hoc-blockchain-lieu-co-muon

Cùng xem state của ngành lập trình hiện giờ để trả lời xem bây giờ học code blockchain liệu có muộn

https://thanhle.blog/en/blog/bay-gio-hoc-blockchain-lieu-co-muon

**Comment by nguyentrungson97** [comment]
em cũng đang là 1 đứa làm react, đang học qua blockchain, đọc bài này có thêm nhiều động lực quá!!!

**Comment by thanhlmm** [comment]
https://tinhte.vn/thread/infographic-thoi-dai-web-3-0-la-gi-co-hoi-cong-viec-trong-thoi-dai-web-3-0.3606346/?ta_from_block=home_featured_threads

Một bài viết hay về cơ hội nghề nghiệp của Web 3.0

**Comment by PhongLi** [comment]
Thời điểm hiện tại thị trường blockchain đã trải qua khá nhiều biến động lớn. Theo anh cơ hội về ngành này và tiềm năng của nó trong tương lai sẽ thế nào ạ, có bị ảnh hưởng quá nhiều bởi các phốt và biến động không ạ?

**Reply by PhongLi** [reply]
Trong đường link tinhte a share ở trên em thấy cũng có khá nhiều ý kiến trái chiều về blockchain trong tương lai dưới comment. Là 1 người có kinh nghiệm làm trong ngành này mong anh cho ý kiến ạ.

**Reply by thanhlmm** [reply]
Đối với anh thì tới vì hype, trend,... nhưng ở lại vì value. Nếu thấy nó có giá trị thì nên ở lại, ko thì rời đi. Còn em chưa rõ ràng thì anh nghĩ em nên làm thử thôi sẽ có câu trả lời cho riêng mình

9 người 10 ý mà :D

**Reply by PhongLi** [reply]
sao anh xóa mất bài này rồi ạ : < Tìm đọc lại mà thấy 404 r
